dmoj/cli.py

```diff
@@ -202,17 +231,19 @@ def execute(self, line):
                 except Exception as io:
                     err = str(io)
             else:
-                src = []
+                file_suffix = '.' + language_id
```

Comment:
This is merely ridiculous. You think `.PY2` is totally a valid extension? Not just `.py2`, but capital `.PY2`.